pandas/tests/io/test_excel.py

```diff
@@ -2065,11 +2065,10 @@ def test_path_local_path(self, merge_cells, engine, ext):
 
 
 @td.skip_if_no('openpyxl')
-@pytest.mark.parametrize("merge_cells,ext,engine", [
-    (None, '.xlsx', 'openpyxl')])
-class TestOpenpyxlTests(_WriterBase):
+@pytest.mark.parametrize("ext", ['.xlsx'])
```

Comment:
Only one test in each of these classes was actually using the `engine` parameter so figured cleaner to just specify there

Reply:
`merge_cells` was also never used but rather required due to the mix of pytest / subclassing paradigms; I've done away with it entirely here